packages/ui5-info/src/ui5-theme-info.ts

```diff
@@ -68,18 +69,26 @@ export function getUi5Themes(ui5Version: string = defaultVersion): UI5Theme[] {
     const ui5VersionSince = ui5Version.replace('snapshot-', '');
     const cleanSemVer = coerce(ui5VersionSince);
     if (cleanSemVer) {
-        return Object.values(ui5Themes).filter((ui5Theme) => {
-            // Check if the theme is within the valid version range
-            const isValidSinceVersion = ui5Theme.sinceVersion ? gte(cleanSemVer, ui5Theme.sinceVersion) : true;
-            const isValidUntilVersion = ui5Theme.untilVersion ? lt(cleanSemVer, ui5Theme.untilVersion) : true;
+        return Object.values(ui5Themes)
+            .map((ui5Theme) => {
+                const theme = { ...ui5Theme };
 
-            // Include themes that are within their version support range
-            if (isValidSinceVersion && isValidUntilVersion) {
-                return true;
-            }
+                if (
+                    theme.id === ui5ThemeIds.SAP_BELIZE &&
```

Comment:
Can we please change the implementation so any theme gets marked as deprecated if its outside the range